src/classes/mech/Mech.ts

```diff
@@ -1045,6 +1046,10 @@ class Mech implements IActor {
     return this.features('Counters')
   }
 
+  public get CountersOnlyMech(): ICounterData[] {
+    return this.features('Counters',false)
```

Comment:
The original features always included things from the pilot, which meant that calling counters for the mech from the pilot caused infinite recursion.